redhat-upgrade-tool.spec

```diff
@@ -84,10 +84,10 @@ install -m 644 examples/boom.conf ${RPM_BUILD_ROOT}/boot/boom
 sed -i 's/enable = False/enable = True/' ${RPM_BUILD_ROOT}/boot/boom/boom.conf
 
 # probably all man here we can skip, but keeping for possible later remove
-mkdir -p ${RPM_BUILD_ROOT}/%{_mandir}/man5
-mkdir -p ${RPM_BUILD_ROOT}/%{_mandir}/man8
-install -m 644 man/man5/boom.5 ${RPM_BUILD_ROOT}/%{_mandir}/man5
-install -m 644 man/man8/boom.8 ${RPM_BUILD_ROOT}/%{_mandir}/man8
+#mkdir -p ${RPM_BUILD_ROOT}/%%{_mandir}/man5
+#mkdir -p ${RPM_BUILD_ROOT}/%%{_mandir}/man8
+#install -m 644 man/man5/boom.5 ${RPM_BUILD_ROOT}/%%{_mandir}/man5
+#install -m 644 man/man8/boom.8 ${RPM_BUILD_ROOT}/%%{_mandir}/man8
```

Comment:
Please keep the code clean. If code is not supposed to be here, it should be removed.